test/dependencies_for_runtests.jl

```diff
@@ -75,7 +75,8 @@ float_types = (Float32, Float64)
 
 include("utils_for_runtests.jl")
 
-archs = test_architectures()
+#archs = test_architectures()
+archs = [CPU(), GPU()]
```

Comment:
Test fail because of this! You hardcoded that `archs` is always both CPU and GPU so then the CI tries to test GPU-related methods on CPU clusters. Can we undo this?

Reply:
Ah!  Great to know and thanks for pointing this out.

I well retry this without the allowscalar on my computer and see what happens.  Thanks!

Reply:
I undid my silly change.